Libs/MRML/CLI/vtkMRMLCommandLineModuleNode.h

```diff
@@ -119,6 +119,12 @@ class VTK_MRML_CLI_EXPORT vtkMRMLCommandLineModuleNode : public vtkMRMLNode
   /// \sa GetStatus(), IsBusy()
   const char* GetStatusString() const;
 
+  void SetOutputText(const std::string& text, bool modify = true);
```

Comment:
Could you document the news methods like what is done for `SetStatus` and `GetStatus` ?